src/otx/algo/anomaly/stfpm.py

```diff
@@ -0,0 +1,46 @@
+"""OTX STFPM model."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from anomalib.models.image.stfpm import Stfpm as AnomalibStfpm
+
+from otx.core.model.entity.base import OTXModel
+from otx.core.model.module.anomaly import OTXAnomaly
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+
+
+class STFPM(OTXAnomaly, OTXModel, AnomalibStfpm):
```

Comment:
To match the anomalib naming, maybe we could rename this to
```suggestion
class Stfpm(OTXAnomaly, OTXModel, AnomalibStfpm):
```